valuesfile/valuesfile.go

```diff
@@ -19,6 +19,10 @@ func NewRenderer(readFile func(filename string) ([]byte, error), basePath string
 	}
 }
```

Comment:
```suggestion
ซ้ำซ้ำของ #
```suggestion
}
```
```suggestion
}
```